integration/service_level_objectives_test.go

```diff
@@ -34,32 +35,38 @@ func TestServiceLevelObjectivesCreateGetUpdateAndDelete(t *testing.T) {
 	assert.Equal(t, expected.Description, actual.Description)
 	assert.True(t, expected.Thresholds.Equal(actual.Thresholds))
 
+	time.Sleep(time.Second)
```

Comment:
Why the sleep? Is it expected that the SLO object is not available through API immediately? Or is it available but incomplete?

Reply:
I found that in a few rare cases, the read replicas were behind just enough to make this fail without some artificial delay.

Reply:
Ok, thanks for the explanation.